test/Microsoft.IdentityModel.Tests/ValidationDelegates.cs

```diff
@@ -29,6 +29,8 @@
 using System.Collections.Generic;
 using System.IdentityModel.Tokens.Jwt;
 using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.Tokens.Saml;
+using Microsoft.IdentityModel.Tokens.Saml2;
```

Comment:
we don't need M.IM.Tokens or M.IM.Tokens.Saml usings.
Another way to return a different token, is to define a CustomSecurityToken like in: CustomTypes.cs. If we use CustomSecurityToken, then we can remove the M.IM.Tokens.Saml2 using.

In fact, this CustomSecurityToken should be moved to DerivedTypes.cs.

Reply:
Moved the CustomSecurityToken into DerivedTypes.cs and removed the unnecessary using statements.